contracts/pool/src/tests/config.rs

```diff
@@ -95,10 +94,9 @@ fn update_config() {
 }
 
 #[test]
-#[should_panic(expected = "Pool: UpdateConfig: Unauthorize")]
+#[should_panic]
```

Comment:
Why did you remove the panic message?

Reply:
once that we got rid of the `sender` parameter, I thought that it's okay just to expect panic. Fixed now, as per your comments from here and yesterday

Reply:
You should never skip the message, because your implementation might panic for whole variety of reasons.